src/chrome/content/rules/kf5.com.xml

```diff
@@ -0,0 +1,25 @@
+<!--
+	kf5.com is an open site, so there are lots of *.kf5.com sites, which are too much to list all of them.
```

Comment:
It seems that they all support https, so a wildcard with exclusions would make sense here.
The current targets can be kept as test urls.